### PR TITLE
parallel() func to do ops in parallel

### PR DESCRIPTION
**Pull request by shreekarSS**
tests/test_ocs_basic_install.py -- Added func parallel() to create_ebs_volumes() to create and attach volumes to the worker node in parallel

Signed-off-by: Shreekar <sshreeka@redhat.com>

### CODE REVIEW

**Comment by shreekarSS** [comment]
with parallel()
install OCS            Minimal installation of OCS                          1837.940152645111               PASSED

without parallel()
install OCS            Minimal installation of OCS                          2035.9336388111115              PASSED

File: tests/test_ocs_basic_install.py [hunk 1]
```diff
@@ -177,16 +178,17 @@ def create_ebs_volumes(
...
-            instance_id=worker['id'],
-            name=f"{worker['name']}_extra_volume",
-            size=size,
-        )
+    with parallel() as p:
+        for worker in worker_instances:
+            log.info(
+                f"Creating and attaching {size} GB volume to {worker['name']}"
+            )
+            p.spawn(aws.create_volume_and_attach,
```
Comment (petr-balogh) [inline tests/test_ocs_basic_install.py]: please keep the indent and brackets like it was, following this [coding guidelines](https://github.com/red-hat-storage/ocs-ci/blob/master/docs/coding_guidelines.md)

Comment (shreekarSS) [reply tests/test_ocs_basic_install.py]: Thanks, updated

File: tests/test_ocs_basic_install.py [hunk 2]
```diff
@@ -177,16 +178,18 @@ def create_ebs_volumes(
...
-            name=f"{worker['name']}_extra_volume",
-            size=size,
-        )
+    with parallel() as p:
+        for worker in worker_instances:
+            log.info(
+                f"Creating and attaching {size} GB volume to {worker['name']}"
+            )
+            p.spawn(
+                    aws.create_volume_and_attach,
```
Comment (petr-balogh) [inline tests/test_ocs_basic_install.py]: too big indent, not 4 spaces but 8

**Comment by clacroix12** [comment]
LGTM once indentation is fixed

File: tests/test_ocs_basic_install.py (hunk 2 above)
Comment (shreekarSS) [reply tests/test_ocs_basic_install.py]: Updated, thanks!

**Comment by vasukulkarni** [comment]
@shreekarSS fix the indentation, If you get confused talk to someone in tier1 ( the examples here should help, if not raise an issue :) - https://github.com/red-hat-storage/ocs-ci/blob/master/docs/coding_guidelines.md

**Pull request merged**